docs/en_US/Compressor/Overview.md

```diff
@@ -180,12 +180,45 @@ class YourQuantizer(nni.compression.tensorflow.Quantizer):
 
     def quantize_weight(self, weight, config, **kwargs):
         """
-        weight is the target weight tensor
-        config is the selected dict object in config_list for this layer
-        kwargs contains op, op_types, and op_name
-        design your quantizer and return new weight
+        quantize should overload this method to quantize weight tensors.
+        This method is effectively hooked to :meth:`forward` of the model.
+
+        Parameters
+        ----------
+        weight : Tensor
+            weight that needs to be quantized
+        config : dict
+            the configuration for weight quantization
         """
         return new_weight
+    
+    def quantize_output(self, output, config, **kwargs):
+        """
+        quantize should overload this method to quantize output.
+        This method is effectively hooked to `:meth:`forward` of the model.
+
+        Parameters
+        ----------
+        output : Tensor
+            output that needs to be quantized
+        config : dict
+            the configuration for output quantization
+        """
```

Comment:
`new_output`  undefine?

Reply:
Oh, I follow the `quantize_weight` format. This means user needs to return a tensor, which is the quantized output.

Reply:
OK. At least add `...`here?

Reply:
Sure. `...` will make more sense.

Reply:
I add a comment 
``` python
# Put your code to generate `new_input` here
return new_input
```
I think this will be more clear.